Controllers/PlayersController.cs

```diff
@@ -6,36 +6,24 @@ namespace PlayersList;
 
 public class PlayersController : Controller
 {
-    // GET /players/PlayerInfo
-    public ActionResult PlayerInfo()
-    {
-        var playersList = new List<Player>() {
+     List<Player> playersList = new List<Player>() {
             new Player(1, "Olive", "Tree", 21, (Teams)1),
             new Player(2, "Ray", "O'Sun", 25, (Teams)2),
             new Player(3, "Rod", "Knee", 20, (Teams)3),
             new Player(4, "Penelope", "Cruz", 31, (Teams)1)
-
-
-                // new Player(){ PlayerID=1, FirstName="Olive", LastName="Tree", Age = 21 },
-                // new Player(){ PlayerID=2, FirstName="Ray", LastName="O'Sun", Age = 25 },
-                // new Player(){ PlayerID=3, FirstName="Rod", LastName="Knee", Age = 20 },
-                // new Player(){ PlayerID=4, FirstName="Penelope", LastName="Cruz", Age = 31 },
             };
 
-        ViewData["players"] = playersList;
-
-        return View(); //the view for *this* controller / action combo
-        // Views/Players/PlayerInfo.cshtml
+    // GET /players/PlayerInfo
+    public ActionResult PlayerInfo()
```

Comment:
The URL for this is composed of the Controller Name + Action Result Method Name, so `/Players/PlayerInfo` - just like we did with the property names on a class, no need to repeat `Player` here twice in the url.  So to fix, we can change the method name here (along with coresponding views).  

The traditional name usually provided here is either `/Players/List` or, assuming the default action when navigating to Players is to view the list, you can name the method `Index` which is assumed - so the url will just be `/Players/`

Reply:
We will handle in #9